ReactAndroid/src/main/java/com/facebook/react/views/image/events/ImageLoadedEvent.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.react.views.image.events;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.uimanager.events.Event;
+import com.facebook.react.uimanager.events.RCTEventEmitter;
+
+public class ImageLoadedEvent extends Event<ImageLoadedEvent> {
+
+  public static final String EVENT_NAME = "topImageLoad";
+
+  public ImageLoadedEvent(int viewId, long timestampMs) {
+    super(viewId, timestampMs);
+  }
+
+  @Override
+  public String getEventName() {
+    return EVENT_NAME;
+  }
+
+  @Override
+  public short getCoalescingKey() {
+    // All events for a given view can be coalesced.
+    return 0;
+  }
+
+  @Override
+  public void dispatch(RCTEventEmitter rctEventEmitter) {
+    rctEventEmitter.receiveEvent(getViewTag(), getEventName(), Arguments.createMap());
```

Comment:
do you plan to actually pass some data here? Otherwise it would be more efficient to send null instead of creating empty map